vignettes/Supporting_additional_objects.Rmd

```diff
@@ -135,21 +149,23 @@ Sharing these functions within a separate package requires an export.
 The simplest way to do this is with Roxygen.
 
 ```{r}
-#' Skimming functions for `sfc_MULTIPOLYGON` objects.
+#' Skimming functions for `sfc` objects.
 #' @export
 skim_sf <- skim_with(
-  sfc_MULTIPOLYGON = sfl(
+  sfc = sfl(
     missing = n_missing,
     n = length,
     n_unique = n_unique,
     valid = ~ sum(sf::st_is_valid(.)),
-    funny = funny_sf
+    crs = get_crs
   )
 )
 
+### This also seems unnecessary
```

Comment:
Same here.

Reply:
Just fYI I responded below about why it's ot correct to say it is unnecessary.

Reply:
Remember that this is showing how to do this in your own package, where you need to export it and you don't want your users having to recreate it all the time.
```suggestion
### This also seems unnecessary
```